File-Formats/DocIO/Working-with-Table-Of-Contents.md

```diff
@@ -588,4 +588,5 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 ## See Also
 
 * [How to change the text of TOC Entries in the Table of content in the Word document?](https://www.syncfusion.com/kb/13019/how-to-change-the-text-of-toc-entries-in-the-table-of-content-in-the-word-document)
-* [How to change tab leader of Table Of Contents in Word document](https://www.syncfusion.com/kb/12659/how-to-change-tab-leader-of-table-of-contents-in-word-document)
\ No newline at end of file
+* [How to change tab leader of Table Of Contents in Word document](https://www.syncfusion.com/kb/12659/how-to-change-tab-leader-of-table-of-contents-in-word-document)
+* [How to customize the TOC entries style in Word document?](https://www.syncfusion.com/kb/13638/how-to-customize-the-toc-entries-style-in-word-document)
```

Comment:
https://www.syncfusion.com/kb/13639/how-to-customize-the-toc-entries-style-in-word-document

Reply:
@MohanaselvamJothi -I have changed it